src/java.management/share/classes/javax/management/MBeanServerFactory.java

```diff
@@ -420,7 +365,7 @@ private static Class<?> loadBuilderClass(String builderClassName)
         }
 
         // No context class loader? Try with Class.forName()
-        return ReflectUtil.forName(builderClassName);
+        return Class.forName(builderClassName);
```

Comment:
Maybe we should keep that call to ReflectUtil here for the moment?

Reply:
The java.management module is mapped to the boot loader so it's okay to replace the use of ReflectUtil.forName with Class.forName here (no behavior change).

Reply:
OK - I withdraw my objection then